Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM crystallang/crystal:1.10.1-alpine
+FROM crystallang/crystal:nightly-alpine
```

Comment:
In general we're not in favor of using nightlies, as they're not stable and can cause sudden breakages. What's the reason you need to be on the nightly track?

Reply:
I werent planning to merge this pr, I am waiting for the 1.11.x release. That version is almost finished (bug fixes are the only thing merged at this point) so I wanted to test compatibility before the release next week.

Reply:
Let me know when you want me to review